ui/packages/shared/utilities/src/bigint.ts

```diff
@@ -15,7 +15,7 @@
  * Divides two bigints and returns a number with two decimal places
  */
 export const divide = (a: bigint, b: bigint): number => {
-  return Number((a * 10000n) / b) / 10000;
+  return Number((a * 1000000n) / b) / 1000000;
```

Comment:
out of curiosity, why did the value being multiplied by change here?

Reply:
Since `bigint` divisions don't give any decimals, we premultiply it by 1M and then divide the result again by 1M to get the result with decimals. 

I increased the number from 10K to 1M to get more precision to the decimal part, which was needed for the `linearScale` function. 

Hope it answers the question.